javascript/javascript1/week4/voiceAssistant.js

```diff
@@ -0,0 +1,75 @@
+let nameC = ""
+let todos = []
+function calc(command){
+    console.log(eval(command.substring(8)))
+}
+function getReply(command){
+    let arr = command.split(' ')
+if(command.includes("Hello")) {
+    let givenName = command.split(' ')
+    nameC=givenName[4]
+    console.log(`Nice to meet you ${nameC}`)
+}
+else if(!isNaN(arr[2])){
+    calc(command);
+}
+else {
+    switch(command){
+    case "What is my name":
+        if(nameC!=="") console.log(`${nameC}`)
+        else console.log("name has not yet been mentioned")
+        break;
+    case "Add fishing to my todo":
+        todos.push("fishing")
+        console.log("fishing added to your todo")
+        break;
+    case "Add singing in the shower to my todo":
+        todos.push("singing in the shower")
+        console.log("singing in the shower added to your todo")
+        break;
+    case "Remove fishing from my todo":
+        for(let i=0;i<todos.length;i++){if(todos[i]=="fishing"){todos.splice(i,1)}}
+        console.log("Removed fishing from your todo")
+        break;
+    case "What is on my todo?":
+        console.log(todos)
+        break;
+    case "What day is it today?":
+        dateArr = new Date().toDateString().split(" ");
```

Comment:
Make sure to assign this with either a `let` or `const`.

Reply:
yeah .. thanks